### ISSUE TITLE
General code clean-up

### ISSUE THREAD

**Issue by rgarmsen2295** [joncatanio/GrannyGauntlet#6]
Admittedly, some of the source code has "lazy spots" (read: a lot of them), and should be cleaned up.  This includes some weird style choices such as:

+ Private variable naming inconsistency
+ Include orderings
+ Unnecessary includes
+ Spacing issues
+ Other naming issues

...and more, add/fix issues as you see them - again, if it's bad (now or in the future) it probably wasn't intentional and should be fixed.

**Comment by rgarmsen2295** [comment]
The private variable one in particular kills me (half are of the form "variable_" the other half are normal).  Was trying something from the gameprogrammingpatterns book and it just led to my class variables looking crazy in spots.  Also getter/setter consistency.

**Comment by joncatanio** [comment]
Haha it's all good we can try to fix when we see them. I would say though we should all make an effort to fix sooner rather than later when changing a single variable may be more difficult.

**Comment by joncatanio** [comment]
Also we should try to convert 'naked' pointers to shared_ptr's if possible. It might be more appropriate to do in C++, if it's not necessary though then we don't need to worry about it.

**Comment by rgarmsen2295** [comment]
Agreed with above, there's an issue specifically about converting the naked pointers, cuz it is such a glaring issue
